migration/v0.9.x-v0.10.x/upgrade-apps.md

```diff
@@ -0,0 +1,36 @@
+
+# Upgrade v0.9.x to v0.10.0
+
+You will need to follow these steps in order to upgrade each Compliant Kubernetes environment from v0.8.x to v0.9.0. The instructions assume that you work from the root of the `compliantkubernetes-apps` repository, has the latest changes already pulled and configured `CK8S_CONFIG_PATH` correctly.
+
+1. Checkout the new release: `git checkout v0.10.0`.
+
+2. Run init to get new defaults: `./bin/ck8s init`
+
+3. Upgrade workload cluster applications
+  ```bash
+  ./bin/ck8s apply wc
+  ```
+
+5. Upgrade service cluster applications
+  ```bash
+
+   # Upgrade Elasticsearch
+   ./bin/ck8s ops helmfile sc -l app=opendistro apply
+
+   # Wait for master pod 0 to be up and for elasticsearch to have started
+
+   # Reload security config
+   ./bin/ck8s ops kubectl sc -n elastic-system exec opendistro-es-master-0 -- chmod +x ./plugins/opendistro_security/tools/securityadmin.sh
+   ./bin/ck8s ops kubectl sc -n elastic-system exec opendistro-es-master-0 -- ./plugins/opendistro_security/tools/securityadmin.sh \
+       -cd plugins/opendistro_security/securityconfig/ \
+       -icl -nhnv \
+       -cacert config/admin-root-ca.pem \
+       -cert config/admin-crt.pem \
+       -key config/admin-key.pem
```

Comment:
This will remove all other (manually added) users like it's done before? Perhaps it could be worth adding a note about that

Reply:
I've changed it so that only roles.yml is reloaded instead of everything.
I've also added a note.

Reply:
Nice, so rolebindings will stay as well as users?

Reply:
Yes